src/libstd/ffi/c_str.rs

```diff
@@ -415,8 +415,7 @@ impl CStr {
     ///
```

Comment:
you need to remove the `use std::str` up on line 410, now that `str` isn't being used anymore.